pkg/core/bundle_read.go

```diff
@@ -14,56 +15,52 @@ func errNotEOF(err error) bool {
 	return err != nil && err.Error() != "EOF"
 }
 
-// ReadAt reads some bundle data with an optional offset. Streamed bundles ignore the offset.
-func (b *Bundle) ReadAt(file *fsEntry, destination []byte, offset int64) (int, error) {
+// ReadAt reads some bundle data with an optional offset
+func (b *Bundle) ReadAt(file *FsEntry, destination []byte, offset int64) (int, error) {
+	logger := b.l.With(
```

Comment:
this is s'th to start doing everywhere, i guess?

use the zap `With()` api to add locals to logger and log a start message before something that could possibly return an error that we need to log, then returning the wrapped error and not logging an error message before the return?

cc. @kerneltime @joaoh821C

Reply:
yes I thought it was a good pattern for readability, trading the extra (minor) cost of storing this logger for less clutter in tracing.

Reply:
follow-up note: comment only. No change requested